xLights/SubModelsDialog.cpp

```diff
@@ -961,21 +986,100 @@ void SubModelsDialog::OnNodesGridLabelLeftClick(wxGridEvent& event)
 
 void SubModelsDialog::OnNodesGridCellRightClick(wxGridEvent& event)
 {
+    wxMenu mnu;
     if (event.GetRow() != -1) {
         //NodesGrid->GoToCell(event.GetRow(), 0);
-        wxMenu mnu;
 
         mnu.Append(SUBMODEL_DIALOG_REMOVE_DUPLICATE, "Remove Duplicates");
-
-        mnu.Connect(wxEVT_MENU, (wxObjectEventFunction)&SubModelsDialog::OnNodesGridPopup, nullptr, this);
-        PopupMenu(&mnu);
-    }
+        mnu.Append(SUBMODEL_DIALOG_ELIDE_DUPLICATE, "Elide Duplicates");
+        mnu.Append(SUBMODEL_DIALOG_SORT_POINTS, "Sort Strand Points Geometrically...");
```

Comment:
what is the difference from Elide to Remove? "Elide" normally means "omit"(I think) so is Elide removing all instances of a node if there are duplicates?

Reply:
For elide, we are leaving a space so that the timing isn't affected.
1,2,2,3 becomes 1,2,3 with remove, or 1,2,,3 with elide.

Reply:
Certainly open to suggestions on a better word for elide, but I took it from a definition I saw, meaning "to suppress" or "to strike out".

Reply:
Give me a different word Gil and I'd be more than happy to do a find/replace for you.  Other people had the same reaction to the use of the word "elide", but they didn't have a suggestion and said it was fine.  Would be great to have a good suggestion.

Reply:
I haven't dug into this so I can't figure out why as programmers having ",0," vs ",," makes anything easier.  We should be able to deal with either.  And does this change work with all existing models.  Some folks will have the old format now does all code need to deal with both versions?

Reply:
The more I think about it, the more I prefer the term "suppress" vs. "elide".  What do you folks think about that change.  Remove duplicates is 1,2,3; suppress duplicates would be 1,2,,3.

Reply:
Yep "suppress" was what I had in mind also when I read your comment about how you found the "elide" term.

Reply:
OK, I changed that.

If you would like any explanation or discussion of anything else, you can always email me (any address at the merryoncherry.org domain goes to me) and we can set up a zoom or something.